16-google/cms.ts

```diff
@@ -0,0 +1,28 @@
+import { Event, normal, metronome, TimedDependency, Timeout } from "../../src";
+
+export class ClusterManagementSoftware extends TimedDependency {
+  public queue1: number = 0;
+  public queue2: number = 0;
+  private queueLimit: number = 2500;
+  constructor(protected timeouts: Timeout[]) {
```

Comment:
Try to depend on a Stage rather than a specific type of stage where possible. The ClusterManagementSoftware never actually cares if it is a timeout or not, so why not type this as a Stage instead.